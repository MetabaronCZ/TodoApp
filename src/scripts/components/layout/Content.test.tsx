import React from 'react';

import { render } from '@testing-library/react';
import { describe, expect, it } from '@jest/globals';

import { withMockedProviders } from 'test/component';
import { Content } from 'components/layout/Content';

describe('components/layout/Content', () => {
  it('should render correctly', () => {
    const tree = render(
      withMockedProviders(
        <Content>
          <div>CONTENT</div>
        </Content>,
      ),
    );
    expect(tree.container).toMatchSnapshot();
    expect(tree.getByText('CONTENT')).toBeInTheDocument();
  });
});
