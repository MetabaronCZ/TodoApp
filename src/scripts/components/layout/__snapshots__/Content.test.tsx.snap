// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/layout/Content should render correctly 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding: 10px 0;
}

<div>
  <main
    class="c0"
  />
</div>
`;

exports[`components/layout/Content should render with children 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding: 10px 0;
}

<div>
  <main
    class="c0"
  >
    <label>
      child
    </label>
  </main>
</div>
`;
